linux/clipboard_watcher_plugin.cc

```diff
@@ -2,16 +2,11 @@
 
 #include <flutter_linux/flutter_linux.h>
 #include <gtk/gtk.h>
-#include <sys/utsname.h>
-
-#include <cstring>
 
 #define CLIPBOARD_WATCHER_PLUGIN(obj)                                     \
   (G_TYPE_CHECK_INSTANCE_CAST((obj), clipboard_watcher_plugin_get_type(), \
                               ClipboardWatcherPlugin))
 
-ClipboardWatcherPlugin* plugin_instance;
```

Comment:
g_signal_connect() & friends allow passing a "user data" pointer to avoid global variables